Assets/MixedRealityToolkit.Services/TeleportSystem/MixedRealityTeleportSystem.cs

```diff
@@ -14,8 +15,16 @@ namespace Microsoft.MixedReality.Toolkit.Services.Teleportation
     /// <summary>
     /// The Mixed Reality Toolkit's specific implementation of the <see cref="Microsoft.MixedReality.Toolkit.Core.Interfaces.TeleportSystem.IMixedRealityTeleportSystem"/>
     /// </summary>
-    public class MixedRealityTeleportSystem : BaseEventSystem, IMixedRealityTeleportSystem
+    public class MixedRealityTeleportSystem : BaseCoreSystem, IMixedRealityTeleportSystem
     {
+        public MixedRealityTeleportSystem(
+            IMixedRealityServiceRegistrar registrar,
+            Transform playspace) : base(registrar, null) // Teleport system does not use a profile
+        {
+            Playspace = playspace;
+            IsInputSystemEnabled = (registrar.GetService<IMixedRealityInputSystem>(showLogs: false) != null);
```

Comment:
>(registrar.GetService<IMixedRealityInputSystem>(showLogs: false) != null) [](start = 35, length = 73)

Just making sure I understand this, this means that in order to create a teleport system, there's an ordering required (i.e. I have to both new up an input system, and also register it, before I can actually new this up). i.e. it's not possible to create this first, then enable an input system, and have things light up after.

Reply:
at this time, the teleport system required an input system. that dependency is not intended to be changed in this PR.